dali/ft/filecopy.hpp

```diff
@@ -72,6 +72,8 @@ class DALIFT_API FileFormat
     bool hasQuote() const                           { return (quote == NULL) || (*quote != '\0'); }
     bool hasQuotedTerminator() const                { return quotedTerminator; }
     const char * getFileFormatTypeString() const        { return FileFormatTypeStr[type]; }
+    bool hasJsonMarkup() const {return (rowTag.length() && *rowTag.get()=='/');} //need to add a more explicit markup indicator
+    bool hasXmlMarkup() const {return (rowTag.length() && *rowTag.get()!='/');}
 
```

Comment:
It will require some work.. retrieving, serialize, deserialize, etc.  I'll look at it.